src/chrome/content/rules/topekazoo.org.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+	Invalid certificate:
+		email.topekazoo.org
+		imap.topekazoo.org
+		mail.topekazoo.org
+		pop.topekazoo.org
+		smtp.topekazoo.org
+		store.topekazoo.org
```

Comment:
WFM

Reply:
Invalid certificate for me, but OK, I'll move it.